build/target-repository/composer.json

```diff
@@ -7,11 +7,15 @@
     ],
     "require": {
         "php": "^7.0|^8.0",
-        "phpstan/phpstan": "^0.9.3 || ^0.12.82"
+        "phpstan/phpstan": "^0.12.83"
```

Comment:
@TomasVotruba it seems will make not installable by require prefixed from php 7

Reply:
after installed on php 7.3, the phpstan should be removed from composer.json to make it installable from php 7

Reply:
Yes, that's correct.

The phpstan for PHP 7.0 must be used from here https://github.com/phpstan/phpstan-src and downgraded to PHP 7.0